app/presenters/content_item_presenter.rb

```diff
@@ -87,7 +87,11 @@ def display_time(timestamp)
   end
 
   def any_updates?
-    DateTime.parse(content_item["public_updated_at"]) != DateTime.parse(content_item["details"]["first_public_at"])
+    if (first_public_at = content_item["details"]["first_public_at"]).blank?
+      false
+    else
+      DateTime.parse(content_item["public_updated_at"]) != DateTime.parse(first_public_at)
+    end
```

Comment:
This conditional feels backwards (you're using the assignment in the `if` in the `else` block.  It might be better to turn this around, and use `.present?` instead of `.blank?`